revolv/static/project.scss

```diff
@@ -439,7 +439,7 @@ $gray-9: #999999;
             }
         }
 
-        .update-text {
+        .update-text p {
```

Comment:
Nah, so the stuff rendered by ckeditor renders to html, and paragraph breaks are rendered as p tags. So then {{ project.description|safe }} will render p tags sometimes, so we need to have to style on those to make it look right!